tools/passwd.bzl

```diff
@@ -0,0 +1,56 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
```

Comment:
tl;dr I think we should add this to `bazelbuild/rules_docker`.

Perhaps under `docker/contrib` for now.  It should be very straightforward to add a rule that uses it under `docker/testdata` and verifies the result under `docker/build_test.sh`.

WDYT?